From 065a4fa3e6c8ea5df6b5d1bf225e2964bc43f333 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20Garc=C3=ADa=20Moreno?= <dani@danigm.net>
Date: Thu, 28 Apr 2022 17:19:58 +0200
Subject: [PATCH] server: Set STATUS_RUNNING just once

Do not set the STATUS_RUNNING when the ZIP_SERVER started, that state is
set now only when the real backend server is running.
---
 kolibri/utils/server.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kolibri/utils/server.py b/kolibri/utils/server.py
index 64a5a9d724..828e3784ab 100644
--- a/kolibri/utils/server.py
+++ b/kolibri/utils/server.py
@@ -334,7 +334,6 @@ class PIDPlugin(SimplePlugin):
 
     def ZIP_SERVING(self, zip_port):
         self.bus.zip_port = zip_port or self.bus.zip_port
-        self.set_pid_file(STATUS_RUNNING)
 
     def EXIT(self):
         try:
-- 
2.35.1

